### PR TITLE
Added ccx enable/disable flag to advance settings and show hide ccx coach option on lms [#94366274]

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
**Background:** In this PR I have added CCX enable setting to Advance settings of course. 

**Studio Updates:** 

  1. User can set CCX enable from studio -> course -> Advance settings -> Enable CCX.
  2. When CCX is disabled on platform, the user will not see ccx_enable field on advance settings.  

**LMS Updates:** 

  1. After enabling setting only then can the user can see CCX Coaches option in drop down under ADMINISTRATION LIST MANAGEMENT inside membership -> instructor dashboard.
  2. On making ccx_enable setting false then ccx coach tab will be hidden from coach

**Testing:** For instructions on enabling CCX and for creating CCX coaches, see https://openedx.atlassian.net/wiki/display/OPEN/CCX+Test+Plan and http://draft-custom-courses.readthedocs.org/en/latest/building_course/custom_courses.html

https://github.com/jazkarta/edx-platform/issues/85
@carsongee @pdpinch  @pwilkins 
screen shot 2015-05-13 at 8 53 15 pm

screen shot 2015-05-12 at 5 34 13 pm

### CODE REVIEW

File: common/lib/xmodule/xmodule/tabs.py [hunk 1]
```diff
@@ -758,7 +758,7 @@ def can_display(self, course, settings, *args, **kw):
         the user is one.
         """
         user_is_coach = False
-        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False):
+        if settings.FEATURES.get('CUSTOM_COURSES_EDX', False) and course.enable_ccx:
```
Comment (pdpinch) [inline common/lib/xmodule/xmodule/tabs.py]: Maybe an obvious question: what happens if course.enable_ccx hasn't been set? Most courses in an already-running instance won't have this value set.

**Comment by pdpinch** [comment]
I think we also needs changes to the tests for the CCX tab and the advanced settings page.

File: common/lib/xmodule/xmodule/tabs.py (hunk 1 above)
Comment (amir-qayyum-arbisoft) [inline common/lib/xmodule/xmodule/tabs.py]: @pdpinch then its value will be False

**Comment by amir-qayyum-arbisoft** [comment]
@pdpinch I am working on test part of PR

**Comment by amir-qayyum-khan** [comment]
Note: I added 'CUSTOM_COURSES_EDX': true, in features inside cms.env.json

@pdpinch @pwilkins

**Comment by amir-qayyum-khan** [comment]
@carsongee  done with testing coverage and quality :sunny: 
cc @pdpinch 

screen shot 2015-05-27 at 3 00 34 pm
screen shot 2015-05-27 at 3 00 14 pm

**Comment by pdpinch** [comment]
I reformatted the cover letter a bit to better match the model at http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/process/cover-letter.html

I also added some links to the docs.

This looks good to go. @amir-qayyum-khan please submit this PR to edX.

**Comment by amir-qayyum-khan** [comment]
sure @pdpinch

**Comment by amir-qayyum-khan** [comment]
Merged on edX platform https://github.com/edx/edx-platform/pull/8207

**Pull request closed without merging**